application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/params/GbfsVehicleRentalDataSourceParameters.java

```diff
@@ -10,11 +10,21 @@ public record GbfsVehicleRentalDataSourceParameters(
   HttpHeaders httpHeaders,
   String network,
   boolean geofencingZones,
-  boolean overloadingAllowed
+  boolean overloadingAllowed,
+  AllowedRentalType allowedRentalType
```

Comment:
Please write a constructor that makes sure this is never null. Like this: https://github.com/opentripplanner/OpenTripPlanner/blob/f4924468169adb1ca188fbf6e73a85135feca7d1/application/src/main/java/org/opentripplanner/routing/alertpatch/AlertUrl.java#L7-L9